### PR TITLE
Make README.md look slightly more epic

### PR DESCRIPTION
**Pull request by worldbeater**
## What does the pull request do?

This PR attempts to make Avalonia README.md file look slightly more epic, by bringing an artwork, consolidating the badges, adding shadows and removing extra borders from the screenshots.

## What is the current behavior?

Currently, there are [huge images in README.md](https://github.com/AvaloniaUI/Avalonia/blob/17e56a8e649b662fb45f3e3093773a3c48c8514f/readme.md). 
I really like how the new theme looks like ❤️, but the images are really huge!

## What is the updated/expected behavior with this PR?

image

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,30 +1,18 @@
...
 
-| Gitter Chat | Build Status (Win, Linux, OSX) | Open Collective | NuGet | MyGet |
-|---|---|---|---|---|
-|  [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/AvaloniaUI/Avalonia?utm_campaign=pr-badge&utm_content=badge&utm_medium=badge&utm_source=badge) | [![Build Status](https://dev.azure.com/AvaloniaUI/AvaloniaUI/_apis/build/status/AvaloniaUI.Avalonia)](https://dev.azure.com/AvaloniaUI/AvaloniaUI/_build/latest?definitionId=4) | [![Backers on Open Collective](https://opencollective.com/Avalonia/backers/badge.svg)](#backers) [![Sponsors on Open Collective](https://opencollective.com/Avalonia/sponsors/badge.svg)](#sponsors) | [![NuGet](https://img.shields.io/nuget/v/Avalonia.svg)](https://www.nuget.org/packages/Avalonia) | [![MyGet](https://img.shields.io/myget/avalonia-ci/vpre/Avalonia.svg?label=myget)](https://www.myget.org/gallery/avalonia-ci) |
+<img src="https://user-images.githubusercontent.com/6759207/84751662-7c79da00-afc5-11ea-8780-dda28db70b76.png" width="700" />
```
Comment (jmacato) [inline readme.md:11]: Let's add a disclaimer here to let users know that Fluent UI theme isnt ready yet.

**Review by jmacato — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (worldbeater) [reply readme.md:11]: Done, thanks!

**Comment by KvanTTT** [comment]
What about social media preview https://github.com/AvaloniaUI/Avalonia/issues/2914? I think this is an ideal time to set it up using [your image](https://user-images.githubusercontent.com/6759207/84750966-ac74ad80-afc4-11ea-97a4-067d99c15a7d.png).

**Comment by worldbeater** [comment]
Thank you @KvanTTT for the friendly reminder!
Planning to improve the image a bit to look more distinct and to fit into opengraph image size.

**Pull request merged**